### ISSUE TITLE
About page > Fix Accessibility link

### ISSUE THREAD

**Issue by jefftriplett** [djangocon/2017.djangocon.us#67]
We reference the `/about/accessibility` page which does not exist. We should create it or reference the right page.

```
- _site/about/index.html
  *  internally linking to /about/accessibility, which does not exist (line 323)
     <a href="/about/accessibility">accessible</a>
```

**Comment by williln** [comment]
Should be addressed in #72.

**Comment by jefftriplett** [comment]
Fixed!
